### ISSUE TITLE
Format command not appearing in palette

### ISSUE THREAD

**Issue by nolanbconaway** [ryantam626/jupyterlab_code_formatter#160]
**Checklist prior to opening an issue**
- [x] I have followed fully the installation steps laid out in the documentation site.
- [x] I have restarted jupyterlab.
- [x] I have read the FAQ section in the documentation site.

**Describe the bug**

Same problem as https://github.com/ryantam626/jupyterlab_code_formatter/issues/43 -- the format command does not appear in the palette. One additional piece of info i have not seen in other issues is this uncaught JS error:

```
Uncaught (in promise) TypeError: Cannot read property 'baseUrl' of undefined
    at a.request (client.js:8)
    at a.getVersion (client.js:28)
    at m.<anonymous> (index.js:144)
    at Generator.next (<anonymous>)
    at index.js:8
    at new Promise (<anonymous>)
    at IWSG.r (index.js:4)
    at m.checkVersion (index.js:142)
    at new m (index.js:36)
    at activate (index.js:166)
```

I have tried the following enable commands but to not change in behavior:

- `jupyter serverextension enable --py jupyterlab_code_formatter`
- `jupyter serverextension enable --py jupyterlab_code_formatter --sys-prefix`
- `jupyter serverextension enable --py jupyterlab_code_formatter --system`

**Diagnostic commands**
Please attach the output of the following commands (please format them properly)

- `pip freeze`

```
appdirs==1.4.4
appnope==0.1.0
astroid==2.4.2
attrs==19.3.0
backcall==0.2.0
better-postgres==2.0.0
betterpy==0.5.0
black==19.10b0
bleach==3.1.5
boto3==1.14.20
botocore==1.17.20
certifi==2020.6.20
chardet==3.0.4
click==7.1.2
cycler==0.10.0
decorator==4.4.2
defusedxml==0.6.0
descartes==1.1.0
docutils==0.15.2
entrypoints==0.3
freezegun==0.3.15
idna==2.10
importlib-metadata==1.7.0
ipykernel==5.3.2
ipython==7.16.1
ipython-genutils==0.2.0
ipywidgets==7.5.1
isort==4.3.21
jedi==0.17.1
Jinja2==2.11.2
jmespath==0.10.0
json5==0.9.5
jsonschema==3.2.0
jupyter==1.0.0
jupyter-client==6.1.6
jupyter-console==6.1.0
jupyter-core==4.6.3
jupyterlab==2.2.0
jupyterlab-code-formatter==1.3.4
jupyterlab-server==1.2.0
kiwisolver==1.2.0
lazy-object-proxy==1.4.3
MarkupSafe==1.1.1
matplotlib==3.2.2
mccabe==0.6.1
mistune==0.8.4
mizani==0.7.1
nbconvert==5.6.1
nbformat==5.0.7
notebook==6.0.3
numpy==1.19.0
packaging==20.4
palettable==3.3.0
pandas==1.0.5
pandocfilters==1.4.2
parso==0.7.0
pathspec==0.8.0
patsy==0.5.1
pexpect==4.8.0
pickleshare==0.7.5
plotnine==0.7.0
prometheus-client==0.8.0
prompt-toolkit==3.0.5
psycopg2-binary==2.8.5
ptyprocess==0.6.0
pydocstyle==5.0.2
Pygments==2.6.1
pylint==2.5.3
pyparsing==2.4.7
pyrsistent==0.16.0
python-dateutil==2.8.1
pytz==2020.1
pyzmq==19.0.1
qtconsole==4.7.5
QtPy==1.9.0
regex==2020.7.14
requests==2.24.0
s3transfer==0.3.3
scipy==1.5.1
Send2Trash==1.5.0
six==1.15.0
snowballstemmer==2.0.0
statsmodels==0.11.1
structlog==20.1.0
terminado==0.8.3
testpath==0.4.4
toml==0.10.1
tornado==6.0.4
traitlets==4.3.3
typed-ast==1.4.1
urllib3==1.25.9
wcwidth==0.2.5
webencodings==0.5.1
widgetsnbextension==3.5.1
wrapt==1.12.1
zipp==3.1.0
```

- `jupyter labextension list`

```
JupyterLab v2.2.0
Known labextensions:
   app dir: /Users/nconaway/.pyenv/versions/3.6.8/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        @ryantam626/jupyterlab_code_formatter v1.3.4  enabled  OK
```

- `jupyter serverextension list`

```
config dir: /Users/nconaway/.jupyter
    jupyterlab_code_formatter  enabled 
    - Validating...
      jupyterlab_code_formatter  OK
config dir: /Users/nconaway/.pyenv/versions/3.6.8/etc/jupyter
    jupyterlab  enabled 
    - Validating...
      jupyterlab 2.2.0 OK
```

**Screenshots**

**Additional context**

**Comment by nolanbconaway** [comment]
I downgraded from jupyterlab 2.2.0 to 2.1.0 and the formatter command is now appearing, so i think this has something to do with the new version!

**Comment by ryantam626** [comment]
Hi! I will have a look this weekend :+1:

**Comment by jchhabra1C** [comment]
Hi! I am having the same issue, can you please help?

- `pip freeze`
`appdirs==1.4.4
appnope==0.1.0
attrs==19.3.0
backcall==0.2.0
beautifulsoup4==4.9.1
black==19.10b0
bleach==3.1.5
branca==0.4.1
certifi==2020.6.20
chardet==3.0.4
click==7.1.2
click-plugins==1.1.1
cligj==0.5.0
coverage==5.0.3
cycler==0.10.0
decorator==4.4.2
defusedxml==0.6.0
descartes==1.1.0
distlib==0.3.1
entrypoints==0.3
filelock==3.0.12
Fiona==1.8.13.post1
Flask==1.1.2
folium==0.11.0
GDAL==3.1.0
geographiclib==1.50
geopandas==0.8.0
geopy==2.0.0
idna==2.10
ipykernel==5.3.2
ipython==7.16.1
ipython-genutils==0.2.0
ipywidgets==7.5.1
itsdangerous==1.1.0
jedi==0.17.1
Jinja2==2.11.2
joblib==0.16.0
json5==0.9.5
jsonschema==3.2.0
jupyter-client==6.1.6
jupyter-contrib-core==0.3.3
jupyter-contrib-nbextensions==0.5.1
jupyter-core==4.6.3
jupyter-highlight-selected-word==0.2.0
jupyter-latex-envs==1.4.6
jupyter-nbextensions-configurator==0.4.1
jupyterlab==2.1.0
jupyterlab-code-formatter==1.3.4
jupyterlab-server==1.2.0
keplergl==0.2.0
kiwisolver==1.2.0
logger==1.4
lxml==4.5.2
MarkupSafe==1.1.1
matplotlib==3.2.2
mistune==0.8.4
munch==2.5.0
nbconvert==5.6.1
nbformat==5.0.7
networkx==2.1
notebook==6.0.3
numpy==1.18.3
osmnx==0.15.1
packaging==20.4
pandas==1.0.3
pandocfilters==1.4.2
parso==0.7.0
pathspec==0.8.0
pexpect==4.8.0
pickleshare==0.7.5
prometheus-client==0.8.0
prompt-toolkit==3.0.5
protobuf==3.12.3
psycopg2==2.8.5
psycopg2-binary==2.8.5
ptyprocess==0.6.0
Pygments==2.6.1
pyparsing==2.4.7
pyproj==2.6.1.post1
pyrsistent==0.16.0
python-dateutil==2.8.1
pytz==2019.3
PyYAML==5.3.1
pyzmq==19.0.1
QtPy==1.9.0
regex==2020.7.14
requests==2.24.0
Rtree==0.9.4
scikit-learn==0.23.1
scipy==1.5.1
seaborn==0.10.1
Send2Trash==1.5.0
Shapely==1.7.0
six==1.14.0
soupsieve==2.0.1
SQLAlchemy==1.3.18
terminado==0.8.3
testpath==0.4.4
threadpoolctl==2.1.0
toml==0.10.1
tornado==6.0.4
tqdm==4.47.0
traitlets==4.3.3
traittypes==0.2.1
typed-ast==1.4.1
urllib3==1.25.9
virtualenv==20.0.26
wcwidth==0.2.5
webencodings==0.5.1
Werkzeug==1.0.1
widgetsnbextension==3.5.1
wntr==0.2.2.1
xlrd==1.2.0
XlsxWriter==1.2.9
xlwt==1.3.0 `
 
- `jupyter labextension list`
`JupyterLab v2.1.0
Known labextensions:
   app dir: /usr/local/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        @ryantam626/jupyterlab_code_formatter v1.3.4  enabled  OK

Uninstalled core extensions:
    @jupyter-widgets/jupyterlab-manager
    @jupyterlab/latex
    @ryantam626/jupyterlab_code_formatter
    jupyter-gmaps
    keplergl-jupyter`

- `jupyter serverextension list`
`config dir: /Users/jchhabra/.jupyter
    jupyterlab_code_formatter  enabled
    - Validating...
      jupyterlab_code_formatter  OK
    jupyter_nbextensions_configurator  enabled
    - Validating...
      jupyter_nbextensions_configurator 0.4.1 OK
config dir: /usr/local/Cellar/python@3.8/3.8.3_2/Frameworks/Python.framework/Versions/3.8/etc/jupyter
    jupyterlab_code_formatter  enabled
    - Validating...
      jupyterlab_code_formatter  OK
config dir: /usr/local/etc/jupyter
    jupyterlab  enabled
    - Validating...
      jupyterlab 2.1.0 OK
    jupyterlab_code_formatter  enabled
    - Validating...
      jupyterlab_code_formatter  OK`

![image](https://user-images.githubusercontent.com/62360783/87827134-13d58580-c82f-11ea-9f1e-e17b8842d755.png)
![image](https://user-images.githubusercontent.com/62360783/87827160-22bc3800-c82f-11ea-88bd-8be416369dfe.png)

**Comment by ryantam626** [comment]
@nolanbconaway the issue with JL 2.2.0 should be fixed with the latest release of the plugin, 1.3.5
 @jchhabra1C you have a different issue, follow [this](https://jupyterlab-code-formatter.readthedocs.io/en/latest/faq.html#why-are-the-commands-not-showing-up-in-the-command-pallete-ii)
